devices/flash-zynq7000/flashdrv.c

```diff
@@ -73,13 +73,22 @@ static int flashdrv_regionFind(addr_t offs, u32 *id)
 }
 
 
-static inline void flashdrv_initCmdBuff(const flash_cmd_t *cmd, addr_t offs)
+static inline void flashdrv_serializeTxCmd(u8 *buff, const flash_cmd_t *cmd, addr_t offs)
```

Comment:
Nitpick, out-of-scope of this PR - `addr_t` is a type for physical address _on the target CPU_, not for addresses in general. `uint32_t` is more appropriate choice for SPI NOR flash address.

Reply:
This change should be done in general API in `devs.h` and applied to other drivers as well.